tests/test_base36.nim

```diff
@@ -0,0 +1,121 @@
+import unittest2
+import ../stew/base36
+
+when defined(nimHasUsed): {.used.}
```

Comment:
all versions of Nim `nim-stew` supports have `used`

Reply:
https://github.com/status-im/nim-stew/compare/3a7e91351f18cde1279dc4a48e73d8fe5d6f95fb..89498a6349addf3205cfefff8642f3d6f0f748c0